styles/styles.css

```diff
@@ -0,0 +1,209 @@
+}
+
+.linav{
+    float: left;
+}
+
+.linav a {
+    display: block;
+    color: black;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+  }
+
+.linav a:hover {
+    background-color: #2c9ab7;
+  }
+
+main{
+    margin: 4rem;
+    align-items: center;
+}
+
+p{
+    line-height: 2;
+}
+
+a{
+    color:#42C1EA;
+    text-decoration:none;
+}
+
+.left
+{
+    float:left;
+    width: 55%;
+    height: auto;
+    padding: 20px;
+    text-align: justify;
+}
+
+.right{
+    float:right;
+    width: 35%;
+    height: auto;
+    padding: 20px;
+    text-align: justify;
+}
+
+blockquote{
+    color:#81E3FD;
+    font-size: x-large;
+}
+
+.pagefooter{
+    background-color:#3a3a3a;
+    text-align:center;
+    color: #ffffff;
+    /* background-image: url(../images/adoptadog-logo.svg); */
+    background-size: cover;
+    width: 100%;
+    clear: both;
+    padding: 40px 0 20px 0;
+    background-color: #093c60;
+}
+
+.ulnavfooter{
+    list-style-type: none;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    background-color: transparent;
+    font-weight: bold;
+}
+
+.linavfooter{
+    padding: 0 5px 0 5px;
+    position: relative;
+    left: 0rem;
+    list-style-type:none;
+    background-color: inherit;
+}
+
+.linavfooter a {
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+  }
+
+.linavfooter a:hover {
+    background-color: #2c9ab7;
+  }
+
+.shelterinfo {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+  }
+  
+  .shelterinfo td, .shelterinfo th {
+    border: 1px solid #ddd;
+    padding: 8px;
+  }
+  
+  .shelterinfo tr:nth-child(even){background-color: #f2f2f2;}
+  
+  .shelterinfo tr:hover {background-color: #ddd;}
+  
+  .shelterinfo th {
+    padding-top: 12px;
+    padding-bottom: 12px;
+    text-align: left;
+    background-color: #2c9ab7;
+    color: white;
+  }
+
+  /*Code for the checkout form*/
+  input[type=text], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+  
+  input[type=submit] {
+    width: 100%;
+    background-color: #4CAF50;
+    color: white;
+    padding: 14px 20px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+  }
+  
+  input[type=submit]:hover {
+    background-color: #45a049;
+  }
+  
+  /* div {
+    border-radius: 5px;
+    background-color: #f2f2f2;
+    padding: 20px;
+  } */
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    /* background-color: #2196F3; */
+    padding: 1rem;
+  }
+  .grid-item {
+    /* background-color: rgba(255, 255, 255, 0.8); */
+    /* border: 1px solid rgba(0, 0, 0, 0.8); */
+    padding: 1rem;
+    font-size: 1rem;
+    text-align: center;
+  }
+
+  .flex-item{
+    width: 20%;
```

Comment:
try using flex basis on width's on flex items

https://css-tricks.com/snippets/css/a-guide-to-flexbox/